lib/passport-wsfed-saml2/saml.js

```diff
@@ -119,7 +124,34 @@ SAML.prototype.validateSignature = function (xml, options, callback) {
   }
 };
 
+SAML.prototype.validateCertExpiration = function (samlAssertion) {
+  // TODO - is this version-dependent?
+  if(samlAssertion.getElementsByTagName("X509Certificate").length > 0){
+    cert = samlAssertion.getElementsByTagName("X509Certificate")[0].textContent;
+  } else {
+    cert = this.options.cert;
+  }
+
+  cert = utils.certToPEM(cert);
+  cert = x509.parseCert(cert)
+
+  let nowDate = new Date();
+
+  if (cert.notBefore > nowDate) {
```

Comment:
We can probably do a between check for these so we don't have two explicit checks + early returns.